core/src/option.cpp

```diff
@@ -43,6 +43,14 @@ option::option(km_core_option_scope s, char16_t const *k, char16_t const *v)
   }
 }
 
+km_core_option_item
+option::release() {
```

Comment:
nit - the code style in this file seems to have open braces on a new line.
Maybe also add a short documententation of the memory complexity😄

Reply:
nvm I see you have it in the header